### PR TITLE
Explore sandbox

### PR DESCRIPTION
**Pull request by liamfallon**

### CODE REVIEW

**Review by lapentad — COMMENTED** [review]
Is the .drawio file supposed to be there?

**Comment by liamfallon** [comment]
@lapentad yes, it's the draw.io source file that the .png was generated from. It's there so that the diagram can be modified later.

File: install-guide/ExploreSandbox.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+├── nephio-webui
+├── porch-dev
+└── resource-backend
+
+```
+<details>
+ <summary>You can check the status of an applied kpt package using a "kpt live status package_dir" command.</summary>
+ 
+```
+sudo kpt live status /tmp/kpt-pkg/nephio-controllers/
```
Comment (electrocucaracha) [inline install-guide/ExploreSandbox.md:53]: [This PR](https://github.com/nephio-project/test-infra/pull/105) will allow running things without `sudo`
```suggestion
kpt live status /tmp/kpt-pkg/nephio-controllers/
```

File: install-guide/ExploreSandbox.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+| Component    | Purpose                                                            |
+| ------------ | -------------------------------------------------------------------|
+| Metal LB     | Load balances requests to the cluster                              |
+| Cert Manager | Used for certificate management                                    |
+| Gitea        | Used to allow creation and management of local git repos by Nephio |
+| Postgres     | Used by Gitea to store repositories                                |
+| Cluster CAPI | Used deploy kind workload clusters                                 |
+| IPAM         | A system used to allocate and manage IP addresses                  |
+| VLAM         | A system used to allocate and manage VLANs                         |
```
Comment (electrocucaracha) [inline install-guide/ExploreSandbox.md:103]:
```suggestion
| VLAN         | A system used to allocate and manage VLANs                         |
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]
Great work @liamfallon, thanks for capturing and describing the process. I just found a typo, besides that everything looks good.

File: install-guide/ExploreSandbox.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+$ kind get clusters
+kind
+```
+</details>
+
+<details>
+<summary>Querying the k8s pods running after the install produces output similar to:</summary>
+
+```
+$ kubectl get pods -A
```
Comment (efiacor) [inline install-guide/ExploreSandbox.md:144]: Slight improvement if you don't need the kube-system pods

kubectl get pods -A --field-selector=metadata.namespace!=kube-system

**Review by efiacor — APPROVED** [review]
Minor suggested edit. Overall looks very helpful.

File: install-guide/ExploreSandbox.md (hunk 1 above)
Comment (liamfallon) [reply install-guide/ExploreSandbox.md]: Done

File: install-guide/ExploreSandbox.md (hunk 2 above)
Comment (liamfallon) [reply install-guide/ExploreSandbox.md]: Done

File: install-guide/ExploreSandbox.md (hunk 3 above)
Comment (liamfallon) [reply install-guide/ExploreSandbox.md]: Done

File: learning.md [hunk 4]
```diff
@@ -28,4 +28,9 @@
 * [OpenAirInterface 5G Core Network Deployment using Helm Charts](https://gitlab.eurecom.fr/oai/cn5g/oai-cn5g-fed/-/blob/master/docs/DEPLOY_SA5G_HC.md): OAI 5G core network have different network functions which can be used invidiually or deployed all together in different combination on a production grade Kubernetes cluster like Openshift or a vanilla kubernetes cluster.
 * [https://github.com/Orange-OpenSource/towards5gs-helm](https://github.com/Orange-OpenSource/towards5gs-helm): Reference helm charts from Orange for the Free5gc core network NFs and for the UERANSIM Radio Access Network
 * [https://github.com/matysiaq/kpt-packages-catalog](https://github.com/matysiaq/kpt-packages-catalog): Custom kpt packages for free5gc and UERANSIM from Orange
-* [https://github.com/matysiaq/nephio-pocs/tree/main/free5gc-operator](https://github.com/matysiaq/nephio-pocs/tree/main/free5gc-operator): An improved f5ee5gc operator from Orange
\ No newline at end of file
+* [https://github.com/matysiaq/nephio-pocs/tree/main/free5gc-operator](https://github.com/matysiaq/nephio-pocs/tree/main/free5gc-operator): An improved f5ee5gc operator from Orange
```
Comment (lapentad) [inline learning.md:31]:
```suggestion
* [https://github.com/matysiaq/nephio-pocs/tree/main/free5gc-operator](https://github.com/matysiaq/nephio-pocs/tree/main/free5gc-operator): An improved free5gc operator from Orange
```

Comment (liamfallon) [reply learning.md]: Done

File: install-guide/ExploreSandbox.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+The following base components are installed on the kind management cluster. Base components are standard components in the k8s ecosystem, which Nephio uses out of the box.
+
+| Component    | Purpose                                                            |
+| ------------ | -------------------------------------------------------------------|
+| Metal LB     | Load balances requests to the cluster                              |
+| Cert Manager | Used for certificate management                                    |
+| Gitea        | Used to allow creation and management of local git repos by Nephio |
+| Postgres     | Used by Gitea to store repositories                                |
+| Cluster CAPI | Used deploy kind workload clusters                                 |
+| IPAM         | A system used to allocate and manage IP addresses                  |
```
Comment (gvbalaji) [inline install-guide/ExploreSandbox.md:102]: IPAM and VLAN backends are more like reference implementations that we use. In real world customers may use something else.

**Review by gvbalaji — COMMENTED** [review]
Thanks @liamfallon for the nice write up. Added couple of comments.

File: install-guide/ExploreSandbox.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+inventory-38069595/customresourcedefinition.apiextensions.k8s.io//amfdeployments.workload.nephio.org is Current: CRD is established
+inventory-38069595/customresourcedefinition.apiextensions.k8s.io//smfdeployments.workload.nephio.org is Current: CRD is established
+inventory-38069595/customresourcedefinition.apiextensions.k8s.io//upfdeployments.workload.nephio.org is Current: CRD is established
+inventory-38069595/namespace//nephio-system is Current: Resource is current
+```
+</details>
+
+## Base Components
+
+The following base components are installed on the kind management cluster. Base components are standard components in the k8s ecosystem, which Nephio uses out of the box.
```
[the remark this answers is not in the document]
Comment (liamfallon) [reply install-guide/ExploreSandbox.md]: @gvbalaji I reworded the document along the lines you suggested.

File: install-guide/ExploreSandbox.md (hunk 5 above)
Comment (liamfallon) [reply install-guide/ExploreSandbox.md]: I updated this as well.

**Comment by liamfallon** [comment]
Could we nudge this on please?

**Pull request merged**